conf.py

```diff
@@ -187,9 +190,17 @@
 # Drop any source link suffix
 html_sourcelink_suffix = ""
 
+# branch from which is started to checkout other branches
+# TODO(anyone) use make_help_scripts/deploy_defines.py to set this variable
+if os.environ.get('BASE_BRANCH_PR') is not None:
+  base_branch = os.environ.get('BASE_BRANCH_PR')
+elif os.environ.get('BASE_BRANCH') is not None:
+  base_branch = os.environ.get('BASE_BRANCH')
+else:
+  base_branch = "master"
 
 # Add branches you want to whitelist here.
-smv_branch_whitelist = r"^(foxy|galactic|humble|iron|master)$"
+smv_branch_whitelist = r"^(foxy|galactic|humble|iron|"+ base_branch + r")$"
 smv_released_pattern = r"^refs/(heads|remotes/[^/]+)/(foxy|galactic|humble|iron).*$"
```

Comment:
Do we still have to whitelist foxy and galactic?

Reply:
we still published it, but mark them as EOL. I'd leave them for a bit longer..

Reply:
I don't change or backport anything to the old distros, just let them be published as they are.

Reply:
Got it 👍🏽👍🏽👍🏽